tools/bpf2c/bpf_code_generator.h

```diff
@@ -186,20 +186,19 @@ class bpf_code_generator
          */
         bpf_code_generator_exception(const unsafe_string& what, size_t offset)
             : std::runtime_error(what.raw() + " at offset " + std::to_string(offset))
-        {
-        }
+        {}
     };
 
     /**
-     * @brief Construct a new bpf code generator object.
+     * @brief Construct a new bpf code generator object from an ELF file.
      *
      * @param[in] stream Input stream containing the eBPF file to parse.
-     * @param[in] c_name C compatible name to export this as.
+     * @param[in] file_name C compatible name to export this as.
```

Comment:
Does BPF byte code always have to be from a file?

Reply:
There are two constructors for this class.  This constructor is called when it's an ELF file.  The other constructor is called when it's not.  See existing line https://github.com/microsoft/ebpf-for-windows/blob/main/tools/bpf2c/bpf_code_generator.cpp#L239 which this is consistent with.

Reply:
Added a comment to this effect.